### PR TITLE
fix(cloud): Add changes to allow switching to different OSes

### PR DESCRIPTION
**Pull request by craciunoiuc**
@nderjung do not merge until we get the good to go

### Prerequisite checklist

  - [ ] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [ ] Tested your changes against relevant architectures and platforms;
  - [ ] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

**Comment by craciunoiuc** [comment]
@mogasergiu

**Review by mogasergiu — APPROVED** [review]
LGTM

Reviewed-by: Sergiu Moga <sergiu@unikraft.io>

File: internal/cli/kraft/cloud/tunnel/tunnel.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 type TunnelOptions struct {
 	TunnelProxyPorts   []string `local:"true" long:"tunnel-proxy-port" short:"p" usage:"Remote port exposed by the tunnelling service(s). (default start port is 4444)"`
 	ProxyControlPort   uint     `local:"true" long:"tunnel-control-port" short:"P" usage:"Command-and-control port used by the tunneling service(s)." default:"4443"`
-	TunnelServiceImage string   `local:"true" long:"tunnel-image" usage:"Tunnel service image" default:"official/utils/tunnel:latest"`
+	TunnelServiceImage string   `local:"true" long:"tunnel-image" usage:"Tunnel service image" default:"official/utils/tunnel-1.0:latest"`
```
Comment (nderjung) [inline internal/cli/kraft/cloud/tunnel/tunnel.go]: Please switch to:
```diff
- official/utils/tunnel-1.0:latest
+ official/utils/tunnel:1.0
```

File: internal/cli/kraft/cloud/volume/import/import.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ type ImportOptions struct {
 	Token string             `noattribute:"true"`
 	Metro string             `noattribute:"true"`
 
-	VolimportImage string `local:"true" long:"image" usage:"Volume import image to use" default:"official/utils/volimport:latest"`
+	VolimportImage string `local:"true" long:"image" usage:"Volume import image to use" default:"official/utils/volimport-1.0:latest"`
```
Comment (nderjung) [inline internal/cli/kraft/cloud/volume/import/import.go]: Ditto

**Review by nderjung — CHANGES_REQUESTED** [review]

**Review by nderjung — APPROVED** [review]
Thanks

**Pull request merged**